pkg/controller/build/controller_test.go

```diff
@@ -18,18 +18,27 @@ package build
 
 import (
 	"errors"
+	"fmt"
 	"strings"
 	"testing"
 	"time"
 
+	"github.com/google/go-cmp/cmp"
```

Comment:
nit: I try to keep imports somewhat tidy, in just three blocks: go stdlib imports, non-`knative/build` imports, and then `knative/build` imports. The second block includes packages from `knative/pkg`, since those are vendored and aren't under our direct control in this repo.

Reply:
Okay I like that approach. I think on Vscode Go plugin re-arranges the imports so I have to manually check it didn't override my arrangement 👍